src/@utils/provider.ts

```diff
@@ -54,22 +54,19 @@ export async function getFileInfo(
   }
 }
 
-export async function fetchMethod(url: string): Promise<AxiosResponse> {
-  const result = await axios.get(url)
+async function fetchMethod(url: string): Promise<Response> {
+  const result = await fetch(url)
   if (!result) {
-    console.error(`Error requesting ${url}`)
-    console.error(`Response message: \n${result}`)
+    LoggerInstance.error(`Error requesting ${url}`)
+    LoggerInstance.error(`Response message: \n${result}`)
     throw result
   }
   return result
 }
 
 export async function isValidProvider(url: string): Promise<boolean> {
   try {
-    const response = await ProviderInstance.isValidProvider(
-      url,
-      fetchMethod(url)
-    )
+    const response = await ProviderInstance.isValidProvider(url, fetchMethod)
```

Comment:
why not just `fetch` , the default one , no extra imports or other functions needed

Reply:
true, just removed `fetchMethod` and used default `fetch`. thanks!